App/Logic/ExposureDetectionLogic.swift

```diff
@@ -373,4 +499,5 @@ extension Logic.ExposureDetection {
       )
     }
   }
+   
```

Comment:
<!--
  0 failure: 
  2 warning:  swiftlint (0.39.2..., swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/ExposureDetectionLogic.swift;
  Line: 502;
-->

- :warning: swiftlint (0.39.2): Don't include vertical whitespace (empty line) before closing braces. (vertical_whitespace_closing_braces)
- :warning: swiftlint (0.39.2): Lines should not have trailing whitespace. (trailing_whitespace)